### PR TITLE
Remove old categories from scripts.en.yml

### PR DESCRIPTION
**Pull request by dmcavoy**
Follow up work to the changes to the Assignment Dropdown.

We are no longer using these categories listed in scripts.en.yml. Instead we are using the category field on Course Offering which has been in and being translated for over 2 months (https://github.com/code-dot-org/code-dot-org/pull/44707). Any pattern matching by Crowdin should have happened already so this removes those old categories that are not longer needed.

### CODE REVIEW

File: dashboard/test/models/script_test.rb [hunk 1]
```diff
@@ -1549,36 +1549,6 @@ class CanViewVersion < ActiveSupport::TestCase
     assert_equal("Waiting for review", feedback2_result[:reviewStateLabel])
   end
 
-  # This test checks that all categories that may show up in the UI have
-  # translation strings.
-  test 'all visible categories have translations' do
```
Comment (davidsbailey) [inline dashboard/test/models/script_test.rb:1554]: just confirming my understanding that this statement is now enforced by the not null requirement here? https://github.com/code-dot-org/code-dot-org/blob/b40e5e08b5218068e5d4da51143fe29b4c48975c/dashboard/app/models/course_offering.rb#L7

**Review by davidsbailey — APPROVED** [review]

**Comment by dmcavoy** [comment]
I just realized the category headers in the Assignment Dropdown are not getting translated. I need to do a step before this PR. Will let you know when its ready to come back to

**Comment by dmcavoy** [comment]
Just checked an the translations are now live for the new way we are translating these so going to merge this now

**Comment by davidsbailey** [comment]
Woohoo!

**Pull request merged**